Jetsnack/app/src/main/java/com/example/jetsnack/ui/components/Grid.kt

```diff
@@ -57,7 +57,7 @@ fun VerticalGrid(
             val columnY = Array(columns) { 0 }
             placeables.forEachIndexed { index, placeable ->
                 val column = index % columns
-                placeable.place(
+                placeable.placeRelative(
```

Comment:
We should probably remove this and use `LazyGrid` but not needed for this PR.

Reply:
Good point!